go/libkb/logout.go

```diff
@@ -190,9 +190,9 @@ func CanLogout(mctx MetaContext) (res keybase1.CanLogoutRes) {
 		}
```

Comment:
few lines above, comment: `LoadHasRandomPw` doesn't exist anymore, let's fix that comment